packages/auto-approve/src/utils-for-pr-checking.ts

```diff
@@ -170,12 +191,20 @@ export function getVersions(
 export function doesDependencyChangeMatchPRTitle(
   versions: Versions,
   dependencyRegex: RegExp,
-  title: string
+  title: string,
+  process: string
 ): boolean {
   let dependencyName;
   const titleRegex = title.match(dependencyRegex);
+
   if (titleRegex) {
     dependencyName = titleRegex[2];
+
+    if (process === 'java-dependency') {
+      if (!dependencyName.includes('com.google.')) {
+        return false;
+      }
+    }
```

Comment:
LGTM - there are some `com.google` artifacts, but they were last released in 2014